### DISCUSSION TITLE
Issues calling a .ini

### DISCUSSION THREAD

**Discussion by juanmantelli** [apache/airflow#30941] [Q&A]
My directory is the following:
dags/
├── my_dag.py
└── etls/
    ├── script.py
    ├── config.py
    └── database/
        └── database.ini

my_dag.py calls the script.py file and then it calls to config.py. This last file has the following line of code:
def config(filename='/home/user/airflow/dags/etls/database/database.ini', section='postgresql'):

And for any reason the process does not find the database.ini. I get the following error:
Exception: Section postgresql not found in the /home/user/airflow/dags/etls/database/database.ini file

I tried to change the path using an inexisting ini for example /home/user/airflow/dags/etls/database/imaginary_database.ini
And I am getting the same error.

I am working with Docker.

What should I do ? There is another way to do this?

Thanks!

**Comment by potiuk** [comment]
You need to load it from the location it is in the docker. You need to find out where your files are mounted. You can also use the usual `__file__` variable in python and find out the relative path to your `.ini` file dynamically. Look up google/stack overflow and you will find many examples on how to do it.
